src/main/java/htsjdk/variant/variantcontext/GenotypeBuilder.java

```diff
@@ -185,14 +186,31 @@ public final void reset(final boolean keepSampleName) {
      * created, althrough the contents of array values like PL should never be modified
      * inline as they are not copied for efficiency reasons.
      *
+     * Note: if attributes are added, the new Genotype will be modified
+     *
      * @return a newly minted Genotype object with values provided from this builder
      */
     public Genotype make() {
-        final Map<String, Object> ea = extendedAttributes == null ? NO_ATTRIBUTES : extendedAttributes;
+        final Map<String, Object> ea = (extendedAttributes == null) ? NO_ATTRIBUTES : extendedAttributes;
         return new FastGenotype(sampleName, alleles, isPhased, GQ, DP, AD, PL, filters, ea);
     }
 
     /**
+     * Create a new Genotype object using the values set in this builder.
+     *
+     * After creation the values in this builder can be modified and more Genotypes
+     * created, although the contents of array values like PL should never be modified
+     * inline as they are not copied for efficiency reasons.
```

Comment:
This comment is inconsistent with the actual code in the method (the PL array is copied)